Automattic/wp-calypso
# Theme Showcase: Use Theme Tiers

Related to https://github.com/Automattic/dotcom-forge/issues/4597
Based on #85044

## Proposed Changes

* Add a `useThemeTier` hook and, as a usage example, insert it into the `ThemeTypeBadge` in the Theme Showcase.

| LiTS | LoTS |
|--------|--------|
| <img width="1351" alt="Screenshot 2023-12-08 at 16 22 53" src="https://github.com/Automattic/wp-calypso/assets/2070010/9eb27000-b1ac-4f07-8137-6cc66093e2bb"> | <img width="1388" alt="Screenshot 2023-12-08 at 16 21 56" src="https://github.com/Automattic/wp-calypso/assets/2070010/703f232b-ce8f-4e64-96f6-ded2c3a37ec1"> | 

Some notes on the UI changes of this PR:

- LiTS, LoTS, and Design Picker.
- Only the **text** of the labels has been changed; the tooltip has been left out as it's too complex for this PR.
- Only a handful of themes have the new tier taxonomy. For now, everything else is labelled as "free" or "included in my plan".

## Testing Instructions



* Enable the `themes/tiers` feature flag.
* Open the Theme Showcase.
* Check the label of the following themes:
  * Annalee and Hevor: should be limited to the Premium plan.
  * Screenplay: should be limited to the Personal plan.
  * Bookix: should be limited to the Business plan, and require an additional subscription.

## Pre-merge Checklist
